friends/src/App.js

```diff
@@ -64,23 +77,82 @@ class App extends Component {
     })
 }
 
+updateFriend = (e)=> {
+  e.preventDefault();
+
+  axios.put(`http://localhost:5000/friends/${this.state.editingId}`, {name: this.state.name, age: this.state.age, email:this.state.email})
+    .then(res => {
+      this.setState({ friends: res.data,
+        name: '',
+        age: '',
+        email: ''
+      });
+      console.log(res)
+    })
+    .catch(err => {
+      console.log(err)
+    })
+
+    this.props.history.push("/")
```

Comment:
I recommend putting the ``` this.props.history.push("/")``` in the ```.then``` so if there was a successful update, then it UI will go to the home page. If you server fails, one can resubmit.